zeppelin-interpreter-shaded/pom.xml

```diff
@@ -33,7 +33,7 @@
   <description>Zeppelin Interpreter Shaded</description>
 
   <properties>
-    <shaded.dependency.prefix>org.apache.zeppelin.shaded</shaded.dependency.prefix>
+    <shaded.dependency.prefix>shaded</shaded.dependency.prefix>
```

Comment:
is it necessary? I believe it was fixed in 3.3.0 by [MSHADE-252](https://issues.apache.org/jira/browse/MSHADE-252)

Reply:
> is it necessary? I believe it was fixed in 3.3.0 by [MSHADE-252](https://issues.apache.org/jira/browse/MSHADE-252)

I can't understand what [MSHADE-252](https://issues.apache.org/jira/browse/MSHADE-252) has to do with relocation of classes. In [MSHADE-252](https://issues.apache.org/jira/browse/MSHADE-252) it is about the correct generation of Java source files. There was even another issue [MSHADE-396](https://issues.apache.org/jira/browse/MSHADE-396) which I opened and closed with the help of @kriegaex.

Reply:
If there is still an issue with relocating into its own sub-package, I would recommend changing it to `shaded.org.apache.zeppelin` instead of a common `shaded`.

Reply:
At the moment there is nothing wrong with your suggested prefix, I have changed this and thank you for your opinion.